tools/go.mod

```diff
@@ -0,0 +1,25 @@
+module github.com/grafana/tempo/tools
+
+go 1.20
+
+require (
+	github.com/psampaz/go-mod-outdated v0.9.0
+	gotest.tools/gotestsum v1.10.0
```

Comment:
q: why gotest.tools, and why not github? imo, I would trust github over gotest.tools to not serve me a bad or compromised verison of gotestsum :)

Reply:
Not really sure, but its in their recommendation for `go install`, so that's what I'd put here.

Reply:
And indeed their module is `gotest.tools` also.

Reply:
Merged, since this wasn't a blocking comment, but if we want to discuss and revisit I'm open to it.